src/api/app/views/webui/user/_edit_account.html.haml

```diff
@@ -0,0 +1,20 @@
+- if feature_enabled?(:responsive_ux)
+  - content_for :actions do
+    - if account_edit_link.present?
+      %li.nav-item
+        = link_to(account_edit_link, class: 'nav-link') do
+          %i.fas.fa-lg.mr-2.fa-plus-square
```

Comment:
What is the reason for not using `fa-user-edit`?

Reply:
None, I'll use it right away